src/Shared/Components/CICDHistory/LogStageAccordion.tsx

```diff
@@ -64,23 +84,31 @@ const LogStageAccordion = ({
 
             {isOpen && (
                 <div className="flexbox-col dc__gap-4">
-                    {logs.map((log: string, logsIndex: number) => (
-                        <LogsItemContainer
-                            // eslint-disable-next-line react/no-array-index-key
-                            key={`logs-${stage}-${startTime}-${logsIndex}`}
-                        >
-                            <span className="cn-4 col-2 lh-20 dc__text-align-end dc__word-break mono fs-14">
-                                {logsIndex + 1}
-                            </span>
-                            <p
-                                className="mono fs-14 mb-0-imp cn-0 dc__word-break lh-20"
-                                // eslint-disable-next-line react/no-danger
-                                dangerouslySetInnerHTML={{
-                                    __html: DOMPurify.sanitize(log),
-                                }}
-                            />
-                        </LogsItemContainer>
-                    ))}
+                    {logs.map((log: string, logsIndex: number) => {
+                        const doesLineContainSearchMatch = `${stageIndex}-${logsIndex}` === searchIndex
```

Comment:
Can we create a method to get index, we can set params as object containing stageIndex, logsIndex